playbook.yml

```diff
@@ -2,17 +2,15 @@
 - hosts: omero-all
   roles:
     - role: ome.postgresql
+      postgresql_install_server: true
```

Comment:
This looked to be quite necessary in molecule. Any ideas why, @manics?

Reply:
It should have always been required, but at some point a bug due to changes in how Ansible handles some variables meant it wasn't. This was fixed in the latest major release of the role.

Reply:
Hmmm.... I missed that. I find it surprising that you'd have to set it to true, i.e. if I was using this role it'd be in order to get the server.

Reply:
We could split the role into `postgresql-server` and `postgresql-client` and get rid of the variable?

Reply:
From https://github.com/ome/ansible-role-postgresql#role-variables and https://github.com/ome/ansible-role-postgresql/blob/4.0.0/defaults/main.yml#L8, I also expected `postgresql-server` to default to `true` if unset but I encountered the same behavior as described here.

Could you expand on the changes in the last major release you were referring to @manics?

Reply:
Yes, previously that prevented the server being installed by default since only the client is needed if your PostgreSQL server is on a different host. At some point something changed (different version of Ansible?) which meant it was ignored, the above commit restored the behaviour. Note the role README was always correct.

A better fix might be to split the postgresql role so the client is in a separate role.

Reply:
See https://github.com/ome/ansible-role-postgresql/pull/19